visualizer/create_DB.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+
+import sqlite3
+
+class database(object):
+    @classmethod
+    def createTB(self):
+        conn = sqlite3.connect('../visualizer/cetune_report.db')
+        print "Opened database successfully";
+
+        conn.execute('''create table tb_report
+                (runid_tr varchar(300) primary key not null,
+                runid int not null,
+                status varchar(15) not null,
+                description varchar(50) not null,
+                opsize varchar(10) not null,
+                optype varchar(20) not null,
+                qd varchar(10) not null,
+                driver varchar(10) not null,
+                snnumber int not null,
+                cnnumber int not null,
+                worker int not null,
+                runtime int not null,
+                iops float not null,
+                bw float not null,
+                latency float not null,
+                sniops float not null,
+                snbw float not null,
+                snlatency float not null
+                );''')
+        print"tb_report created successfully."
+
+        conn.close()
+        print "Closed database successfully";
+
+    @classmethod
+    def insert_to_TB(self,data):
+        rowdata = data
+        conn = sqlite3.connect('../visualizer/cetune_report.db')
```

Comment:
I am thinking , db is better located at the same dir with results , let's say under /mnt/data ?